src/appkit.cs

```diff
@@ -3837,6 +3837,10 @@ public interface NSColorList : NSSecureCoding {
 
 		[Export ("removeFile")]
 		void RemoveFile ();
+
+		[Mac (10,11)]
+		[Export ("writeToURL:error:")]
+		bool WriteToURL ([NullAllowed] NSUrl url, [NullAllowed] out NSError errPtr);
```

Comment:
-> `WriteToUrl` (3 letters acronym rule)
-> `errPtr` is quite ugly (even if it match Apple's headers) -> `error`